website/source/docs/providers/docker/r/registry.html.mdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "docker"
+page_title: "Docker: docker_registry"
+sidebar_current: "docs-docker-resource-registry"
+description: |-
+  Logs to a Docker registry, downloads and exports a Docker image.
```

Comment:
Is this description accurate? It seems like maybe this is the description from `docker_image`.